.github/workflows/auto-author-assign.yml

```diff
@@ -0,0 +1,15 @@
+# .github/workflows/auto-author-assign.yml
+name: Auto Author Assign
+
+on:
+  pull_request:
+
+permissions:
+  pull-requests: write
+
+jobs:
+  assign-author:
+    runs-on: ubuntu-latest
+    if: ${{ github.repository == 'software-mansion/starknet.py' }}  # do not run on forks
+    steps:
+      - uses: toshimaru/auto-author-assign@v1.6.2
```

Comment:
Looks fine I think, what I'd change is fixing `@version` to a specific version of the commit for extra security, [see here how it's done](https://github.com/software-mansion/starknet.py/blob/development/.github/workflows/dependabot.yml#L26)

Reply:
We should do it for every action, at the very least not official ones